app/src/main/java/com/keylesspalace/tusky/adapter/TabAdapter.kt

```diff
@@ -32,14 +32,16 @@ import kotlinx.android.synthetic.main.item_tab_preference.view.*
 
 interface ItemInteractionListener {
     fun onTabAdded(tab: TabData)
+    fun onTabRemoved(position: Int)
     fun onStartDelete(viewHolder: RecyclerView.ViewHolder)
     fun onStartDrag(viewHolder: RecyclerView.ViewHolder)
     fun onActionChipClicked(tab: TabData)
 }
 
 class TabAdapter(private var data: List<TabData>,
                  private val small: Boolean = false,
-                 private val listener: ItemInteractionListener? = null) : RecyclerView.Adapter<TabAdapter.ViewHolder>() {
+                 private val listener: ItemInteractionListener,
+                 private var removeButtonEnabled: Boolean? = null) : RecyclerView.Adapter<TabAdapter.ViewHolder>() {
```

Comment:
Better make `removeButtonEnabled` non-nullable (so its a primitive on the jvm and not an object, also no need for the nullcheck)